src/test/xmeta/CMakeLists.txt

```diff
@@ -5,17 +5,32 @@ project(test_xmeta)
 # using /MD flag for antlr4_runtime (for Visual C++ compilers only)
 set(ANTLR4_WITH_STATIC_CRT ON)
 
+include_directories(
+    ${CMAKE_SOURCE_DIR}/tool/xmeta/
+)
+
 add_executable(test_xmeta "")
 
+target_link_libraries(test_xmeta 
+    WindowsApp.lib
+    comsuppw.lib
+)
+
+
 if (EXISTS "${CMAKE_SOURCE_DIR}/test/xmeta/example/midl3test")
 add_custom_command(TARGET test_xmeta POST_BUILD
     COMMAND ${CMAKE_COMMAND} -E copy_directory
     "${CMAKE_SOURCE_DIR}/test/xmeta/example/midl3test"
     $<TARGET_FILE_DIR:test_xmeta>/midl3test)
 endif()
 
-target_sources(test_xmeta
-    PUBLIC pch.cpp testlexer.cpp testmidlcompat.cpp test_symbol_table.cpp xlang_test_listener.cpp)
+target_sources(test_xmeta PUBLIC
+    pch.cpp
+    testlexer.cpp
+    testmidlcompat.cpp
+    testxmetaemit.cpp
+    xlang_test_listener.cpp
+)
 
 target_include_directories(test_xmeta
     PUBLIC ${XLANG_LIBRARY_PATH} ${XLANG_TEST_INC_PATH} ${CMAKE_CURRENT_SOURCE_DIR}/../output/component/source)
```

Comment:
Is ..output/component/source still being used?

Reply:
Apparently not.